youtube/guide-for-beginners.md

```diff
@@ -1,137 +1,96 @@
 
-* Check the [documentation](https://github.com/inotia00/revanced-documentation) for the other methods/PC guide.
+* Check the [YT troubleshooting page](https://www.reddit.com/r/revancedextended/wiki/yt-troubleshooting/#wiki_issues_with_patching_.26amp.3B_installation) if you get stuck during this guide.
 
-[‎ ](LineBreak)
 
 # **1. Requirements:**
 
-* Your device must be running Android 8.0+
-
-* Your device must use the arm64-v8a architecture. You will be able to check what your device's architecture is at a later point in the tutorial. 
-
-#### **Downloading MicroG, ReVanced Manager, and the YouTube APK...**
+* Your device must be running Android 8.0 or newer.
 
-* Vanced MicroG `(latest version)`:
+* Your device must use the arm64-v8a architecture. (You will find out if it is in a moment) 
 
-**Download and install the Vanced MicroG APK from [here](https://github.com/TeamVanced/VancedMicroG/releases/latest). Or you can download an updated forked version from [inotia00's GitHub](https://github.com/inotia00/VancedMicroG/releases/latest).**
+#### **Downloading MicroG, YouTube APK, & the RVX Manager**
 
-* RVX Manager `(latest)`:
+**a)** If you do not have Vanced MicroG installed, download and install [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases/latest). Or you can use an updated forked version from [inotia00's GitHub](https://github.com/inotia00/VancedMicroG/releases/latest).
 
-**Download and install the RVX Manager APK from the [RvX Manager GitHub page here](https://github.com/inotia00/revanced-manager/releases/latest). The version linked here is the fork of the official Manager made by Inotia00 to be compatible with ReVanced Extended patches.**
+**b)** Download, BUT DO NOT INSTALL the **YouTube** `v18.25.40` (`nodpi`) APK from [this page](https://www.apkmirror.com/apk/google-inc/youtube/youtube-18-25-40-release/youtube-18-25-40-android-apk-download/).
 
-* YouTube APK `(v18.25.40, nodpi)`:
+**c)** Download and install the **RVX Manager** APK from [here](https://github.com/inotia00/revanced-manager/releases/latest). It is inotia00's fork of the official Manager made to be compatible with ReVanced Extended patches.
 
-**Download, BUT DO NOT INSTALL the YouTube `v18.25.40 (nodpi)` APK from [this link](https://www.apkmirror.com/apk/google-inc/youtube/youtube-18-25-40-release/youtube-18-25-40-android-apk-download/).**
+(*If you get an error saying* "App not installed as app isn't compatible with your device." *or* "App not installed.", *it likely means that your device is not supported by the RVX Manager. Refer to the [YT troubleshooting page](https://www.reddit.com/r/revancedextended/wiki/yt-troubleshooting/#wiki_issues_with_patching_.26amp.3B_installation) for further instructions.*)
 
-[‎ ](LineBreak)
 
-# **2. Additional compatibility checks with the RVX Manager:**

```

Comment:
this section was removed bec i confirmed that it wont install  at all if its not armv8